Extras/HistoryView/UIKit/COTrackViewController.m

```diff
@@ -107,7 +107,7 @@ - (UIColor *)suggestedColorForNode: (id <COTrackNode>)node
 
 - (UITableViewCell *)makeCellForNode: (id <COTrackNode>)node
 {
-    static NSString *TrackCellIdentifier = @"TrackCell";
+    static const NSString *const TrackCellIdentifier = @"TrackCell";
```

Comment:
My two cents: I would remove the `const`s since (1) it would be consistent with other objects being non-`const`, and (2) making them `const` makes a promise that the compiler doesn't enforce. But for now I'll be adding `const`s in these kinds of situations just so that when you guys make a decision I can see where they all are in the commits.